src/cli/cli.cpp

```diff
@@ -2091,6 +2091,35 @@ otError Interpreter::ProcessIpAddr(Arg aArgs[])
         {
             OutputIp6AddressLine(*otThreadGetMeshLocalEid(GetInstancePtr()));
         }
+        else if (aArgs[0] == "-v")
```

Comment:
Wonder if we can reuse the `if (aArgs[0].IsEmpty())` case ?

I mean we can consume `-x` and set `verbose` flag if `-x` is specified.
The code would be like:

```c++
    bool verbose = false;

    if (aArgs[0] == "-x") {
      aArgs++;
      verbose = true;
    }
    if (aArgs[0].IsEmpty())
    {
        const otNetifAddress *unicastAddrs = otIp6GetUnicastAddresses(GetInstancePtr());
        for (const otNetifAddress *addr = unicastAddrs; addr; addr = addr->mNext)
        {
            OutputIp6Address(addr->mAddress);
            if (verbose) {
                OutputFormat(" Origin:");
                ....
            }
            OutputLine("");
        }
    }

```

does that make sense? @zwx1995esp